### PR TITLE
arkh chain.json

### PR DESCRIPTION
**Pull request by vincadian**

### CODE REVIEW

**Comment by sunnya97** [comment]
Hi, do you have any explorer available for this chain (does not have to be mintscan)?

Also, do you have at least one seed node we can add to this list?

**Comment by sunnya97** [comment]
There seems to be no code available at the repo you linked.

https://github.com/vincadian/arkh-blockchain

**Review by sunnya97 — DISMISSED** [review]
See above

**Comment by vincadian** [comment]
> There seems to be no code available at the repo you linked.
> 
> https://github.com/vincadian/arkh-blockchain

Hi, the github repo is https://github.com/vincadian/arkh-blockchain/tree/master, because I can't push to main ( still trying to figure why actually)

**Comment by vincadian** [comment]
> Hi, do you have any explorer available for this chain (does not have to be mintscan)?
> 
> Also, do you have at least one seed node we can add to this list?

I emailed mintscan but still no response

**Comment by sunnya97** [comment]
Hey @vincadian, just so I can understand correctly, is this a testnet or a mainnet?

**Comment by vincadian** [comment]
> Hey @vincadian, just so I can understand correctly, is this a testnet or a mainnet?

It is a mainnet

**Comment by vincadian** [comment]
> Hey @vincadian, just so I can understand correctly, is this a testnet or a mainnet?

Ping explorer asked 10 explorer to be listed as a mainnet

**Comment by vincadian** [comment]
Something wrong ?

File: arkh/arkh.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                "address": "https://asc-blockchain-api.arkhadian.com/",
+                "provider": "arkhnetwork"
+            }
+        ]
+    },
+    "explorers": [
+        {
+            "kind": "Ping",
+            "url": "https://testnet.ping.pub/arkh",
+            "tx_page": "https://testnet.ping.pub/arkh/blocks"
```
Comment (sunnya97) [inline arkh/arkh.json:62]: This is the blocks url, not the txs url.  If there is no txs page, that's okay, just remove this line

File: arkh/arkh.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        "seeds": [
+            {
+                "id": "808f01d4a7507bf7478027a08d95c575e1b5fa3c",
+                "address": "asc-dataseed.arkhadian.com:26656"
+            },
+            
+        ],
+        "persistent_peers": [
+           
+        ]
```
Comment (sunnya97) [inline arkh/arkh.json:40]: Please remove if there are none right now.

File: arkh/arkh.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        "fee_tokens": [
+            {
+                "denom": "uarkh",
+                "fixed_min_gas_price": 0.000000001
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/vincadian/arkh-blockchain",
+
```
Comment (sunnya97) [inline arkh/arkh.json:28]:
```suggestion
```

File: arkh/arkh.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "Arkh",
```
Comment (sunnya97) [inline arkh/arkh.json:3]:
```suggestion
    "chain_name": "arkh",
```

File: arkh/arkh.json (hunk 1 above)
Comment (sunnya97) [reply arkh/arkh.json:62]: I think the ping.pub tx url is something like this:

`https://ping.pub/osmosis/tx/${txHash}`

Can you confirm that your explorer has one like this

File: arkh/arkh.json (hunk 2 above)
Comment (sunnya97) [inline arkh/arkh.json:40]:
```suggestion
        ]
```

Comment (sunnya97) [reply arkh/arkh.json:40]: Remove persistent peers if there are none.

File: arkh/arkh.json (hunk 1 above)
Comment (sunnya97) [reply arkh/arkh.json:62]:
```suggestion
            "tx_page": "https://testnet.ping.pub/arkh/tx/${txHash}"
```

File: arkh/arkh.json [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "uarkh",
+                "fixed_min_gas_price": 0.000000001
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/vincadian/arkh-blockchain",
```
Comment (sunnya97) [inline arkh/arkh.json:27]: @vincadian need to know which branch/release to use.  Please cut a release for the version of the code to be used on mainnet.

File: arkh/arkh.json [hunk 6]
```diff
@@ -18,13 +18,24 @@
     "fees": {
         "fee_tokens": [
             {
-                "denom": "uarkh",
+                "denom": "arkh",
```
Comment (sunnya97) [inline arkh/arkh.json:21]: this needs to be the base_denom, which I assume is `uarkh`.

Speaking of which, can you please add an assetlist as well?

See the Osmosis example:  https://github.com/cosmos/chain-registry/blob/master/osmosis/assetlist.json

File: arkh/assetlist.json [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+    "$schema": "../assetlist.schema.json",
+    "chain_id": "arkh",
+    "assets": [
+      {
+        "description": "The native token of Arkhadian",
+        "denom_units": [
+          {
+            "denom": "uarkh",
+            "exponent": 0,
+            "aliases": []
```
Comment (sunnya97) [inline arkh/assetlist.json:11]:
```suggestion
            "exponent": 0
```

File: arkh/assetlist.json [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+          },
+          {
+            "denom": "arkh",
+            "exponent": 6,
+            "aliases": []
```
Comment (sunnya97) [inline arkh/assetlist.json:16]:
```suggestion
            "exponent": 6
```

File: arkh/assetlist.json [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+          }
+        ],
+        "base": "uarkh",
+        "name": "Arkh",
+        "display": "arkh",
+        "symbol": "ARKH",
+        "logo_URIs": {
+          "png": "https://www.arkhadian.com/static/images/600-prism_ejAjDRh.jpg",
```
Comment (sunnya97) [inline arkh/assetlist.json:23]: This is a jpg, not png.  Do you have a png?

File: arkh/assetlist.json [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
```
Comment (sunnya97) [inline arkh/assetlist.json:24]:
```suggestion
```

File: arkh/assetlist.json [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+        ],
+        "base": "uarkh",
+        "name": "Arkh",
+        "display": "arkh",
+        "symbol": "ARKH",
+        "logo_URIs": {
+          "png": "https://www.arkhadian.com/static/images/600-prism_ejAjDRh.jpg",
+          
+        },
+        "coingecko_id": "arkhadian"
```
Comment (sunnya97) [inline arkh/assetlist.json:26]:
```suggestion
        }
```

If not on coingecko yet, please don't add

**Comment by vincadian** [comment]
I am going to update a png

**Comment by sunnya97** [comment]
Wanna keep the black background?  I think it looked cooler that way 😄

File: arkh/arkh.json [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/vincadian/arkh-blockchain",
+        
+        "recommended_version": "v1.0.0",
+        "compatible_versions": [
+            "v1.0.0",
+
```
Comment (sunnya97) [inline arkh/arkh.json:32]:
```suggestion
```

File: arkh/arkh.json [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+        ]
+    },
+    
+    "apis": {
+        "rpc": [
+            {
+                "address": "https://asc-dataseed.arkhadian.com/",
+                "provider": "arkhnetwork"
+            },
+
```
Comment (sunnya97) [inline arkh/arkh.json:56]:
```suggestion
            }
```

File: arkh/arkh.json [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+           
+        }
+    },
+    "peers": {
+        "seeds": [
+            {
+                "id": "808f01d4a7507bf7478027a08d95c575e1b5fa3c",
+                "address": "asc-dataseed.arkhadian.com:26656"
+            },
+
```
Comment (sunnya97) [inline arkh/arkh.json:46]:
```suggestion
            }
```

File: arkh/arkh.json (hunk 12 above)
Comment (sunnya97) [inline arkh/arkh.json:32]:
```suggestion
            "v1.0.0"
```

File: arkh/arkh.json [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+        "git_repo": "https://github.com/vincadian/arkh-blockchain",
+        
+        "recommended_version": "v1.0.0",
+        "compatible_versions": [
+            "v1.0.0"
+        ],
+        "binaries": {
+            "linux/amd64": "https://github.com/vincadian/arkh-blockchain/releases/download/latest/arkh-blockchain_latest_linux_amd64.tar.gz",       
+            "darwin/amd64": "https://github.com/vincadian/arkh-blockchain/releases/download/latest/arkh-blockchain_latest_darwin_amd64.tar.gz",
+
```
Comment (sunnya97) [inline arkh/arkh.json:36]:
```suggestion
            "darwin/amd64": "https://github.com/vincadian/arkh-blockchain/releases/download/latest/arkh-blockchain_latest_darwin_amd64.tar.gz"
```

File: arkh/arkh.json [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+    },
+    "peers": {
+        "seeds": [
+            {
+                "id": "808f01d4a7507bf7478027a08d95c575e1b5fa3c",
+                "address": "asc-dataseed.arkhadian.com:26656"
+            }
+        ]
+    },
+
```
Comment (sunnya97) [inline arkh/arkh.json:47]:
```suggestion
```

File: arkh/assetlist.json [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+        ],
+        "base": "uarkh",
+        "name": "Arkh",
+        "display": "arkh",
+        "symbol": "ARKH",
+        "logo_URIs": {
+          "png": "https://www.arkhadian.com/static/images/600-prism.png",
+        }
+      },
+
```
Comment (sunnya97) [inline arkh/assetlist.json:25]:
```suggestion
      }
```

File: arkh/assetlist.json [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+            "denom": "arkh",
+            "exponent": 6
+          }
+        ],
+        "base": "uarkh",
+        "name": "Arkh",
+        "display": "arkh",
+        "symbol": "ARKH",
+        "logo_URIs": {
+          "png": "https://www.arkhadian.com/static/images/600-prism.png",
```
Comment (sunnya97) [inline arkh/assetlist.json:22]:
```suggestion
          "png": "https://www.arkhadian.com/static/images/600-prism.png"
```

**Pull request merged**